### PR TITLE
Create general start page, Closes #200

### LINKED ISSUE #200: Create general start page
Parent issue: #199

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #200

### CODE REVIEW

File: start.md [hunk 1]
```diff
@@ -1,7 +1,113 @@
 # Lisk Documentation
 
-- landing page
+Welcome to the Lisk documentation!
+
+Beside the **Lisk Protocol**, you find all you need to **setup, run and configure** the different parts of the **Lisk ecosystem**.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
Here you will find all you need to **setup, run and configure**  a Lisk Node and join the network. 
We also describe the **Lisk Protocol**  and **Lisk products**.
```

File: start.md [hunk 2]
```diff
@@ -1,7 +1,113 @@
...
+
+## The Lisk Protocol
```
Comment (MaciejBaj) [inline start.md]: Move to the bottom.

File: start.md [hunk 3]
```diff
@@ -1,7 +1,113 @@
...
+
+The [Lisk Protocol](../lisk-protocol/introduction.md) is a formal set of rules of the Lisk blockchain.
+
+It consists of rules about the P2P communication, block and transaction formats, the consensus algorithm, and of rules how signatures and IDs are computed and verified.
+
```
Comment (MaciejBaj) [inline start.md:10]: Try referring to Lisk Academy chapters.

File: start.md [hunk 4]
```diff
@@ -1,7 +1,113 @@
...
+The protocol defines these rules without focus on concrete implementation, and it is therefore possible to implement it in any compatible programming language.
+
+Currently, the official implementations of the Lisk protocol are [Lisk Core](../lisk-core/introduction.md) and [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md), which are written in Javascript and Typescript.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
Currently, the official implementations of the Lisk protocol are [Lisk Core](../lisk-core/introduction.md) and [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md), which are written in Typescript and [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) written in Javascipt.
```

File: start.md [hunk 5]
```diff
@@ -1,7 +1,113 @@
...
+
+## The Lisk ecosystem
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
## Lisk products
```

File: start.md [hunk 6]
```diff
@@ -1,7 +1,113 @@
...
+
+The Lisk Ecosystem consists of a set of modular software products that provide developers with all the tools they need to develop their own custom blockchain applications.
+It also consists of software that is needed to maintain and monitor the Lisk Main network.
+
+Tool                                                         | Description
+------------------------------------------------------------ | --------------------------------------------------
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) | Modular set of building blocks with specific blockchain related functionality.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
[Lisk SDK](../lisk-sdk/introduction.md) |  Set of modular software products that provide developers with all the tools they need to develop their own custom blockchain applications.
[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) | Modular set of building blocks with specific, blockchain-related, functionality.
```

File: start.md [hunk 7]
```diff
@@ -1,7 +1,113 @@
...
+
+It consists of rules about the P2P communication, block and transaction formats, the consensus algorithm, and of rules how signatures and IDs are computed and verified.
+
+The protocol defines these rules without focus on concrete implementation, and it is therefore possible to implement it in any compatible programming language.
+
+Currently, the official implementations of the Lisk protocol are [Lisk Core](../lisk-core/introduction.md) and [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md), which are written in Javascript and Typescript.
+
+## The Lisk ecosystem
+
+The Lisk Ecosystem consists of a set of modular software products that provide developers with all the tools they need to develop their own custom blockchain applications.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
```

File: start.md [hunk 8]
```diff
@@ -1,7 +1,113 @@
...
+It also consists of software that is needed to maintain and monitor the Lisk Main network.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
```

File: start.md [hunk 9]
```diff
@@ -1,7 +1,113 @@
...
+
+Tool                                                         | Description
+------------------------------------------------------------ | --------------------------------------------------
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) | Modular set of building blocks with specific blockchain related functionality.
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md)   | JS library for interacting with the network.
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) | CLI-tool to interact with the network.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) | Command line tool to interact with the network and maintain a node.
```

File: start.md [hunk 10]
```diff
@@ -1,7 +1,113 @@
...
+
+## The Lisk ecosystem
+
+The Lisk Ecosystem consists of a set of modular software products that provide developers with all the tools they need to develop their own custom blockchain applications.
+It also consists of software that is needed to maintain and monitor the Lisk Main network.
+
+Tool                                                         | Description
+------------------------------------------------------------ | --------------------------------------------------
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) | Modular set of building blocks with specific blockchain related functionality.
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md)   | JS library for interacting with the network.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md)   | Set of libraries implementing Lisk protocol, contains all of the cryptography tools you may need to implement Lisk network interactions into your application.
```

File: start.md [hunk 11]
```diff
@@ -1,7 +1,113 @@
...
+[Lisk SDK](../lisk-sdk/introduction.md) > [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) | CLI-tool to interact with the network.
+[Lisk Hub](../lisk-hub/introduction.md)                      | UI for users to interact with the network, e.g. to manage their accounts, or to send or receive tokens.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
[Lisk Hub](../lisk-hub/introduction.md)                      | Lisk wallet and UI tool for users to interact with the network, e.g. to manage their accounts, send or receive tokens or vote for delegates.
```

File: start.md [hunk 12]
```diff
@@ -1,7 +1,113 @@
...
+
+### Maintain a node
+
+You want to become a node operator?
+
+Great!
+
+There are several good reasons for it:
+
+- Have your **private entry point** to [communicate](../lisk-core/api) with the network
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
- Have your **private entry point** to [communicate](../lisk-core/api) with the network, it's important if you are an **exchange** implementing LSK tokens.
```

File: start.md [hunk 13]
```diff
@@ -1,7 +1,113 @@
...
+Beside this, the **Lisk Framework** offers [Components](../lisk-sdk/lisk-framework/introduction.md).
+Components consist of functionalities that are useful for most Modules, and provide an easy to use interface to perform actions on the application as a whole.
+<br>The [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) initializes the Modules and controls their communication between each other and with the application as a whole.
+To do this, each Module creates its' own communication channel, where it can publish events that it wants to share with the whole application.
+Modules can passively listen to certain events that have been published by other Modules, or they can actively ask for information from other Modules.
+
+4. Interact with your network<br>
+While your network is up and running, use [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) to interact with the network via the CLI (Command-Line-Interface).
+
+5. Build the frontend<br>
```
Comment (MaciejBaj) [inline start.md]: Remove that part for nm
```suggestion
```

File: start.md [hunk 14]
```diff
@@ -1,7 +1,113 @@
...
+For convenience, you may reuse the packages of [Lisk Elements](../lisk-sdk/lisk-elements/introduction.md) which provide related functionalities for creating blockchain applications, like cryptography.
+Beside this, the **Lisk Framework** offers [Components](../lisk-sdk/lisk-framework/introduction.md).
+Components consist of functionalities that are useful for most Modules, and provide an easy to use interface to perform actions on the application as a whole.
+<br>The [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md) initializes the Modules and controls their communication between each other and with the application as a whole.
+To do this, each Module creates its' own communication channel, where it can publish events that it wants to share with the whole application.
+Modules can passively listen to certain events that have been published by other Modules, or they can actively ask for information from other Modules.
+
+4. Interact with your network<br>
+While your network is up and running, use [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) to interact with the network via the CLI (Command-Line-Interface).
```
Comment (MaciejBaj) [inline start.md]: Include a note of liskhq/cryptohraphy implemeting all of the needed crypto libraries

File: start.md [hunk 15]
```diff
@@ -1,7 +1,113 @@
...
+
+1. Setup the Framework<br>
+To get started in building your own blockchain application, set up the [Lisk Framework](../lisk-sdk/lisk-framework/introduction.md).
+The Modules come with a set of predefined default configuration, that let you start and test out your own network immediately.
+
+2. Configure the Modules<br>
+After the first successful start, feel free to configure, exchange or extend the Modules, suited to your specific needs.
+
+3. Write own Modules<br>
+It is also possible to create own custom Modules from scratch, if needed.
```
Comment (MaciejBaj) [inline start.md]: Mention architecture and following the interfaces specs so the modules can communicate

File: start.md [hunk 16]
```diff
@@ -1,7 +1,113 @@
...
+- Registered Delegates, who would like forge actively.
+- Users who do not trust different sources and want to be in full control.
+
+To become a node operator, you need to setup [Lisk Core](../lisk-core/introduction.md) on a server and connect it to the desired network.
+
+### Build a blockchain application
+
+#### What is a blockchain application?
+
+Blockchain applications are applications that communicate with a blockchain network in the background.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
Blockchain applications communicate within a network to agree upon which data is valid and should be stored and which should be rejected and bad actors potentially punished. The set of blockchain rules is already implemented by Lisk modules and libraries. By re-using them and implementing well-designed Lisk SDK Framework it's easy to compose new case-specific applications. Explore more great content of [Blockchain Business](https://lisk.io/academy/blockchain-business) in [Lisk Academy](https://lisk.io/academy).
```

File: start.md [hunk 17]
```diff
@@ -1,7 +1,113 @@
...
+This blockchain network can be used as decentralized database layer to create, read, update or delete data within the system.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
```

File: start.md [hunk 18]
```diff
@@ -1,7 +1,113 @@
...
+
+A **pure blockchain application** has no central data storage at all, and all information that is needed to perform its actions is actually stored within the blockchain itself.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
```

File: start.md [hunk 19]
```diff
@@ -1,7 +1,113 @@
...
+
+A **hybrid blockchain application** may have certain data stored inside of a blockchain, but might additionally use centralized data storage solutions in certain areas.
```
Comment (MaciejBaj) [inline start.md]:
```suggestion
```

File: start.md [hunk 20]
```diff
@@ -1,7 +1,113 @@
...
+
+#### And why do I need it?
```
Comment (MaciejBaj) [inline start.md]: Remove for now, Lisk Academy describes it in-depth.

File: start.md [hunk 21]
```diff
@@ -1,7 +1,113 @@
...
+Modules can passively listen to certain events that have been published by other Modules, or they can actively ask for information from other Modules.
+
+4. Interact with your network<br>
+While your network is up and running, use [Lisk Commander](../lisk-sdk/lisk-commander/introduction.md) to interact with the network via the CLI (Command-Line-Interface).
+
+5. Build the frontend<br>
+When you have the network up and running as desired, it is time to build a nice frontend for your application, so that users are happy to use it.
+
+### Interact with the network
+
```
Comment (MaciejBaj) [inline start.md:39]: Explore the packages of Elements and describe Lisk Client.

**Review by MaciejBaj — CHANGES_REQUESTED** [review]
Please address the left suggestions.

File: start.md (hunk 14 above)
Comment (Tschakki) [reply start.md]: Why here?

File: start.md [hunk 22]
```diff
@@ -1,7 +1,126 @@
...
+There are several ways how to participate in the Lisk project:
+
+ How                       | Where
+-------------------------  | -------------------------------------------------------------------------------------------------------------------
+Contribute to the protocol | Discuss future changes of the protocol with the Science team on [research.lisk.io](https://research.lisk.io/).
+Contribute to the codebase | [LiskHQ Github](https://github.com/LiskHQ) is the place to be! Be sure to follow our [Contribution Guidelines](https://github.com/LiskHQ/lisk-sdk/blob/development/docs/CONTRIBUTING.md) before submitting any code.
+Give feedback              | For general feedback, please use [Lisk on Reddit](https://www.reddit.com/r/Lisk/) or [Lisk Chat](https://lisk.chat/), and [Gitter](https://gitter.im/LiskHQ/lisk) for technical questions and feedback.
+
+## The Lisk Protocol
```
Comment (MaciejBaj) [inline start.md]: Before the "Contribute" section.

File: start.md [hunk 23]
```diff
@@ -1,7 +1,50 @@
...
+
+Action                                                            | Description                                                                                                               
+ ---------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ 
+[Build a blockchain application](build-blockchain-app.md)         | <PRE>1. Alpha SDK [step by step guide](build-blockchain-app.md#how-to-build-a-blockchain-application-with-the-lisk-sdk)<br>  a) How to [create](build-blockchain-app.md#1-setup-the-sdk) your custom blockchain<br>  b) How to [configure](build-blockchain-app.md#2-configure-and-customize-the-application) your blockchain<br>  c) How to create a [custom transaction](build-blockchain-app.md#2b-design-custom-transaction-types)<br>  d) Build the [frontend](build-blockchain-app.md#3-build-the-frontend) and [interact](interact-with-network.md) with the network<br>2. [Why blockchain?](https://lisk.io/academy/blockchain-basics/use-cases)</PRE>
```
Comment (MaciejBaj) [inline start.md]: Chapter names are not consistent with the file names, please do so and use the following titles:
1. How to set up and start Lisk SDK?
2. How to customise blockchain parameters to suit an application use-case?
3. How to add new custom transaction types to implement an application use-case?
4. How to interact with the network?

File: start.md [hunk 24]
```diff
@@ -1,7 +1,50 @@
...
+[Maintain a node](maintain-node.md)                               | <PRE>1. [Deploy](maintain-node.md#how-to-set-up-a-node) your first blockchain node in 5 minutes<br>2. Maintain your existing \[[Binary](../lisk-core/administration/binary.md)&#124;[Docker](../lisk-core/administration/docker.md)&#124;[Source](../lisk-core/administration/source.md)\] node<br>3. Benefits of operating a node.</PRE>
```
Comment (MaciejBaj) [inline start.md]: Same as above - please unify the section file names and titles. Also, make the full sentences being the link here - "How to set up a node"

File: start.md [hunk 25]
```diff
@@ -1,7 +1,50 @@
...
+[Interact with the network](interact-with-network.md)             | Interact with the blockchain network through our [API](https://lisk.io/documentation/lisk-core/api) and install useful [tools](interact-with-network.md#a-use-the-command-line).
+[Get to know the Lisk Protocol](../lisk-protocol/introduction.md) | Learn the underlying logic and technical concepts behind the Lisk Protocol including its [consensus algorithm](../lisk-protocol/consensus.md), [security](../lisk-protocol/security.md), [blocks](../lisk-protocol/blocks.md), [transactions](../lisk-protocol/transactions.md) and [peer communications](../lisk-protocol/p2p-communication.md).
+
+## Lisk products
+
+For each product we provide a general introduction, step-by-step How-Tos and detailed specifications.
+
+Tool                                                         | Description
+------------------------------------------------------------ | --------------------------------------------------
+[Lisk SDK](../lisk-sdk/introduction.md)                      |  Set of modular software products that provide developers with all the tools they need to develop their own custom blockchain applications.
```
Comment (MaciejBaj) [inline start.md]: In descriptions please reflect the GitHub READMEs: https://github.com/LiskHQ/lisk-sdk

**Review by MaciejBaj — CHANGES_REQUESTED** [review]

File: resources.md [hunk 26]
```diff
@@ -0,0 +1,31 @@
+# Resources
+
+- [LiskHQ Github](https://github.com/LiskHQ) - Source code of all software products of Lisk.
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
- [LiskHQ Github](https://github.com/LiskHQ) - Source code for all software products of Lisk.
```

File: resources.md [hunk 27]
```diff
@@ -0,0 +1,31 @@
...
+- [LiskHQ NPM packages](https://www.npmjs.com/~lisk) - List of all published NPM packages of LiskHQ.
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
- [LiskHQ NPM packages](https://www.npmjs.com/~lisk) - Lists all NPM packages published by LiskHQ.
```

File: resources.md [hunk 28]
```diff
@@ -0,0 +1,31 @@
...
+- [Community Tools](https://docs.google.com/spreadsheets/d/1EJ2ni5LBBNM43cCFkvQ7lYyAHeGm_cFwOQkfAqd-fQc/edit#gid=0) -  List of community built tools, wallets and scripts to help make your life easier.
+- [Help Center](https://lisk.io/help-center) - A searchable database of frequently asked questions.
+- [Lisk Academy](https://lisk.io/academy) - A comprehensive and unbiased educational platform on blockchain technology. 
+- [Lisk Improvement Proposals](https://github.com/LiskHQ/lips) - A collection of proposed improvements to the Lisk protocol
+- [Roadmap](https://lisk.io/roadmap) - Explore the development roadmap as we make progress towards the SDK
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
- [Roadmap](https://lisk.io/roadmap) - Explore Lisk's development roadmap. 
```

File: resources.md [hunk 29]
```diff
@@ -0,0 +1,31 @@
+# Resources
+
+- [LiskHQ Github](https://github.com/LiskHQ) - Source code of all software products of Lisk.
+- [LiskHQ NPM packages](https://www.npmjs.com/~lisk) - List of all published NPM packages of LiskHQ.
+- [Community Tools](https://docs.google.com/spreadsheets/d/1EJ2ni5LBBNM43cCFkvQ7lYyAHeGm_cFwOQkfAqd-fQc/edit#gid=0) -  List of community built tools, wallets and scripts to help make your life easier.
+- [Help Center](https://lisk.io/help-center) - A searchable database of frequently asked questions.
+- [Lisk Academy](https://lisk.io/academy) - A comprehensive and unbiased educational platform on blockchain technology. 
+- [Lisk Improvement Proposals](https://github.com/LiskHQ/lips) - A collection of proposed improvements to the Lisk protocol
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
- [Lisk Improvement Proposals](https://github.com/LiskHQ/lips) - A collection of proposed improvements to the Lisk protocol.
```

**Review by RachBLondon — CHANGES_REQUESTED** [review]

File: resources.md [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+- [Community Tools](https://docs.google.com/spreadsheets/d/1EJ2ni5LBBNM43cCFkvQ7lYyAHeGm_cFwOQkfAqd-fQc/edit#gid=0) -  List of community built tools, wallets and scripts to help make your life easier.
+- [Help Center](https://lisk.io/help-center) - A searchable database of frequently asked questions.
+- [Lisk Academy](https://lisk.io/academy) - A comprehensive and unbiased educational platform on blockchain technology. 
+- [Lisk Improvement Proposals](https://github.com/LiskHQ/lips) - A collection of proposed improvements to the Lisk protocol.
+- [Roadmap](https://lisk.io/roadmap) - Explore Lisk's development roadmap. 
+- [News](https://blog.lisk.io/) - Links to the latest news about Lisk, updated weekly.
+
+## Community
+
+Join the community!
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
Join our community!
```

File: resources.md [hunk 31]
```diff
@@ -0,0 +1,31 @@
...
+
+Below are the most essential links to the official Lisk community channels.
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
Below are the essential links to the Lisk community channels.
```

File: start.md [hunk 32]
```diff
@@ -1,7 +1,50 @@
...
+[Interact with the network](interact-with-network.md)             | Interact with the blockchain network through the [API](https://lisk.io/documentation/lisk-core/api) and install useful [tools](interact-with-network.md#a-use-the-command-line) that facilitate the process.
+[Get to know the Lisk Protocol](../lisk-protocol/introduction.md) | Learn the underlying logic and technical concepts behind the Lisk Protocol including its [consensus algorithm](../lisk-protocol/consensus.md), [security](../lisk-protocol/security.md), [blocks](../lisk-protocol/blocks.md), [transactions](../lisk-protocol/transactions.md) and [peer communications](../lisk-protocol/p2p-communication.md).
+
+## Lisk products
+
+For each product we provide a general introduction, step-by-step How-Tos and detailed specifications.
```
Comment (RachBLondon) [inline start.md]:
```suggestion
For each product we provide a general introduction, step-by-step guides and detailed specifications.
```

File: start.md [hunk 33]
```diff
@@ -1,7 +1,50 @@
...
+[Maintain a node](maintain-node.md)                               | <PRE>1. [How to set up a node](maintain-node.md#how-to-set-up-a-node) in 5 minutes<br>2. Maintain your existing \[[Binary](../lisk-core/administration/binary.md)&#124;[Docker](../lisk-core/administration/docker.md)&#124;[Source](../lisk-core/administration/source.md)\] node<br>3. [Benefits of operating a node](maintain-node.md#why-to-set-up-a-node).</PRE>
+[Interact with the network](interact-with-network.md)             | Interact with the blockchain network through the [API](https://lisk.io/documentation/lisk-core/api) and install useful [tools](interact-with-network.md#a-use-the-command-line) that facilitate the process.
```
Comment (RachBLondon) [inline start.md]:
```suggestion
[Interact with the network](interact-with-network.md)             | Interact with the blockchain network through its [API](https://lisk.io/documentation/lisk-core/api) and install useful [tools](interact-with-network.md#a-use-the-command-line) to facilitate the process.
```

**Review by RachBLondon — DISMISSED** [review]

File: resources.md [hunk 34]
```diff
@@ -0,0 +1,31 @@
...
+- [Roadmap](https://lisk.io/roadmap) - Explore Lisk's development roadmap. 
+- [News](https://blog.lisk.io/) - Links to the latest news about Lisk, updated weekly.
+
+## Community
+
+Join our community!
+
+Below are the essential links to the Lisk community channels.
+
+- [Subscribe to the Lisk newsletter](https://mailchi.mp/lisk/newsletter) - Updates straight to your inbox
```
Comment (RachBLondon) [inline resources.md]:
```suggestion
- [Subscribe to our Lisk newsletter](https://mailchi.mp/lisk/newsletter) - Updates straight to your inbox
```

**Review by RachBLondon — APPROVED** [review]

**Pull request merged**